linux/vhba_hot_add_remove/run_iozone_test.sh

```diff
@@ -92,7 +94,9 @@ function run_iozone()
     cd "$testdir_path"
     echo "Run iozone on $part_path" || continue
     # Run iozone testing
-    if [ $(echo "$part_size > 128"|bc) -eq 1 ]; then
+    if [[ "$os_distribution" =~ FreeBSD ]]; then
+        ${IOZONE_PATH} -Ra -g 128K -i 0 -i 1 -b "$iozone_file"
+    elif [ $(echo "$part_size > 128"|bc) -eq 1 ]; then
```

Comment:
It looks like the unstable result is not caused by iozone testing size. So here can change back to 128M.